spark-k8s/Dockerfile

```diff
@@ -61,6 +61,17 @@ COPY --chown=${STACKABLE_USER_UID}:0 spark-k8s/hbase-connectors/stackable/patche
 COPY --chown=${STACKABLE_USER_UID}:0 spark-k8s/hbase-connectors/stackable/patches/${HBASE_CONNECTOR} /stackable/src/spark-k8s/hbase-connectors/stackable/patches/${HBASE_CONNECTOR}
 
 RUN <<EOF
+
+# IMPORTANT: HBase connectors don't support Spark 4 yet, so we skip the build.
```

Comment:
Instead of hiding this deep in a Dockerfile I would prefer to do it similar to what we had for Trino in the past:
https://github.com/stackabletech/docker-images/blob/e021cb5b685d0963c262d3fcee1969671701aafc/trino/Dockerfile#L42-L44

If the `HBASE_CONNECTOR` env var is set via version.py, it is installed. If not not

Reply:
Thanks for checking, I trust you with that